### DISCUSSION TITLE
Is it possible to create custom queries?

### DISCUSSION THREAD

**Discussion by ashwaniarya** [callstack/react-native-testing-library#753] [General]
I am working on a project where we are thinking to create our own custom queries to find the same node used in various components being tested. Although **DOM Testing Library** does support the creation of custom queries but I couldn't find the way to do the same in **react-native-testing-library**.

**Comment by thymikee** [comment]
Not yet, but it's almost there: https://github.com/callstack/react-native-testing-library/pull/573

**Reply by mdjastrzebski** [reply]
I've created a discussion for this topic here #971. Please feel free to comment.
